Havit/Havit/Global/UIComponent/ContentsCollectionViewCell.swift

```diff
@@ -0,0 +1,167 @@
+//
+//  ContentsCollectionViewCell.swift
+//  Havit
+//
+//  Created by 박예빈 on 2022/01/12.
+//
+
+import UIKit
+import SnapKit
+import RxSwift
+
+class ContentsCollectionViewCell: UICollectionViewCell {
+    static var cellID = "ContentsCollectionViewCell"
+    
+    var mainImgView: UIImageView!
+    var alarmImgView: UIImageView!
+    var titleLabel: UILabel!
+    var subtitleLabel: UILabel!
+    var dateLabel: UILabel!
+    var linkLabel: UILabel!
+    var moreButton: UIButton!
+    var isReadImgView: UIImageView!
+    var alarmLabel: UILabel!
```

Comment:
P2;
```suggestion
    var mainImgView = UIImageView()
    var alarmImgView = UIImageView()
    var titleLabel = UILabel()
    var subtitleLabel = UILabel()
    var dateLabel = UILabel()
    var linkLabel = UILabel()
    var moreButton = UIButton()
    var isReadImgView = UIImageView()
    var alarmLabel = UILabel()
```

!로 사용해서 강제 옵셔널 해제를 시키는 것보다는 이런 식으로 해주는게 좋을 것 같습니다.

Reply:
이러면 밑에 setUpCell 함수에서 할당하는것까지 더해 객체를 두번할당하게 되지 않나요?

Reply:
```swift
 var button : UIButton = {
        let btn = UIButton()
        btn.setTitle("다음 뷰로", for: .normal)
        btn.backgroundColor = .red
        return btn
    }()

이렇게 하는거랑 어떤게 더 좋은지 몰라 고민했는데 처음 정의할 때 할당해주는게 강제 옵셔널 해제를 시켜주지 않아서 좋은 방법이었군여! 😃
```

Reply:
@beansbin 저도 수진님 말에 동의 합니다 ! 두번 할당해줄 필요는 없을 것 같습니다 !

Reply:
@sujinnaljin 
밑에 setUpCell 부분에 UILabel를 설정하면서 객체를 할당할 수 있는 코드로 코드리뷰 달았습니다!
https://github.com/TeamHavit/Havit-iOS/pull/28#discussion_r783897222